docs/Spatial.md

```diff
@@ -1,6 +1,92 @@
 #Phidget Spatial Sensor
-More commonly known as an accelerometer.
+The PhidgetSpatial library makes for intuitive and lightning fast development without any compromise. All sensors on this board should be primed for accurate use, here are the [acceleration](http://www.phidgets.com/docs/Accelerometer_Primer), [gyroscopic](http://www.phidgets.com/docs/Gyroscope_Primer), and [magnetic field](http://www.phidgets.com/docs/Compass_Primer) primer guides. For a quick start into your Spatial project see this [basic PhidgetSpatial example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js).
```

Comment:
In the future we should consider creating a simpler version of what you linked to and having it in our documentation.